docker/Dockerfile

```diff
@@ -121,9 +121,9 @@ RUN echo "Adding odc-dependencies" \
 USER root
 ## Patch env when needed here
 # override hdstats - install faster version from dea packages
-RUN echo "Patching python env - hdstats" \
-  && pip install --force-reinstall \
-  --extra-index-url="https://packages.dea.ga.gov.au" hdstats==0.1.8.post1
+# RUN echo "Patching python env - hdstats" \
+#   && pip install --force-reinstall \
+#   --extra-index-url="https://packages.dea.ga.gov.au" hdstats==0.1.8.post1
```

Comment:
can we remove this commented code please

Reply:
sure